### DISCUSSION TITLE
Improve emoji picker in Workspaces

### DISCUSSION THREAD

**Discussion by PSYRJT** [zen-browser/desktop#3788] [Ideas]
Right now there's not any option to look for a specific emoji to set as Workspace icon, so add a search bar or something.
<img width="316" alt="image" src="https://github.com/user-attachments/assets/d1cc4383-51d8-42d4-99ca-bb87c429a4f5" />
anything is better than looking for the emoji you want to use scrolling through this chaotic mess of emojis, the size of them doesn't help either

**Comment by pinage404** [comment]
Duplicates of https://github.com/zen-browser/desktop/discussions/2617
